libbeat/processors/actions/decode_json_fields_test.go

```diff
@@ -190,13 +191,12 @@ func getActualValue(t *testing.T, config *common.Config, input common.MapStr) co
 		logp.LogInit(logp.LOG_DEBUG, "", false, true, []string{"*"})
 	}
 
-	p, err := newDecodeJSONFields(*config)
+	p, err := newDecodeJSONFields(config)
 	if err != nil {
 		logp.Err("Error initializing decode_json_fields")
 		t.Fatal(err)
 	}
 
-	actual, err := p.Run(input)
-
-	return actual
+	actual, _ := p.Run(&beat.Event{Fields: input})
```

Comment:
the error has never been checked for. In this PR I'm trying to adjust interfaces, not fix/improve/change behavior in tests or other pieces.